lib/bolt/cli.rb

```diff
@@ -140,6 +140,10 @@ def create_option_parser(results)
                 "(Optional, defaults to 100)") do |concurrency|
           results[:concurrency] = concurrency
         end
+        opts.on('-t', '--timeout TIMEOUT', Integer,
```

Comment:
Should this be called `timeout` or `connection_timeout`?  I would expect we're able to eventually implement some sort of `execution_timeout` (on the WinRM side the functionality is already there, just not exposed)

Reply:
👍  to `connection_timeout` as it's fairly common to also expose `read_timeout`

Reply:
Probably agree with `connection_timeout`. How often does this need to be used?

Reply:
I think we should set low sensible defaults (1 second seems reasonable to me), making `connection_timeout` rarely used in cases where network connectivity is a problem.